static/src/js/util/url/__tests__/projectCollections.test.js

```diff
@@ -201,7 +202,7 @@ describe('url#decodeUrlParams', () => {
         }
       }
 
-      expect(decodeUrlParams('?p=!collectionId1!collectionId2&pg[1][m]=harmony&pg[1][ets]=f')).toEqual(expectedResult)
+      expect(decodeUrlParams('?p=!collectionId1!collectionId2&pg[1][m]=harmony&pg[1][ets]=f&pg[1][ess]=f')).toEqual(expectedResult)
```

Comment:
This test is for decoding the `ets` parameter, don't include `ess` in the URL, it is unnecessary input that   adds clutter to the test